### DISCUSSION TITLE
Backoffice Custom Data Validation

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#316] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

latest

### Description

Hi all,

I am trying to build a Backoffice page with the following features:

1 - It reads data from a GET HTTP request that respects the crud-service interface
2 - Data can have some fields whose values are in the wrong format
3 - If so, the HTTP GET request returns some error fields along with the standard fields (e.g. suppose the data model has two fields "name" and "surname", the GET request will return "name", "nameError", "surname", "surnameError")
4 - If at least one "error" field exists, then the table row in the backoffice page must be red
5 - If the user clicks on a red row, the edit modal shows the errors close to its related field

So far, I got to cover all the points above except for number 5.
Is there a way to do it with the "bk-form-modal" or any other bk-components?

Thanks in advance.

**Comment by clabene** [comment, marked as answer]
Hi @Lucascanna,

unfortunately, point 5 cannot be achieved with back-kit components as of now.
Fields validation is handled automatically, and no configuration is available to customize this behavior.
